### ISSUE TITLE
my account suspended plz help

### ISSUE THREAD

**Issue by strange3r** [SuperGO2/supergo2-issues#492]
am sorry i didnt know that i cant have multi accounts is there a chance to get my account back ty

**Comment by boydtech761** [comment]
Yes you can, I’ll greatly advice you message https://t.me/boydtech on telegram, he’s the best on account recoveries
